docs/network_disruption/prio.md

```diff
@@ -152,6 +152,43 @@ We first filter for all packets related to `gateway IP` and `node IP` and send t
 
 Finally, we apply a filter to enqueue all packets to class `1:4` whenever the `destination IP` is encompassed by the `hosts` field (see [this documentation](../../docs/network_disruption/hosts-and-services.md) for more details). In this case, a filter is applied for `10.0.1.254/32` and another for `10.0.1.255/32`. If no hosts were specified, a single filter is applied for `0.0.0.0/0` and no traffic is ends up in class `2:1`.
 
+### Network Disruption implementation for pod level with eBPF filters
+
+Tc can use eBPF filters in order to intercept packets at the application layer and allow the access to the payload of packets.
+Network disruption used eBPF filters to apply disruption by filtering with `method` and/or `path`:
+
+Now, let us take the following pod level disruption spec with new filters:
+```
+spec:
+  level: pod
+  selector:
+    app: demo
+  count: 1
+  network:
+    hosts:
+      - 10.0.1.254/31
+    port: 80
+    protocol: tcp
+    method: get <-- eBPF filter
```

Comment:
Are these both HTTP specific? We don't ever seem to mention that in the docs or names or code.

Reply:
Yes, **method** and **path** are related to HTTP. Something like that can do the job?

``` yaml
spec:
  level: pod
  selector:
    app: demo
  count: 1
  network:
    hosts:
      - 10.0.1.254/31
    port: 80 <--- L4 
    protocol: tcp
    http:
      method: get <-- eBPF filter
      path: /test <-- eBPF filter
 ```

Reply:
Yes, I think that is much clearer